src/sbd-inquisitor.c

```diff
@@ -784,52 +784,49 @@ parse_device_line(const char *line)
     int last = 0;
     int max = 0;
     int found = 0;
+    int skip_space = 1;
```

Comment:
Personally I'd prefer bool/true/false for readability (reader knows at a glance what the allowed values are and what they mean)

Reply:
Things like this here and the size_t stuff above are always a hard decision.
Either one goes with how the code in the vicinity looks like, leading to consistent code short term or one goes with more restrictive guide-lines in the hope that over time all the ugly stuff would age out.